.github/workflows/update-browser-version.yml

```diff
@@ -29,6 +29,10 @@ jobs:
         uses: peter-evans/create-pull-request@v5
         with:
           token: ${{ secrets.BROWSER_AUTOMATION_BOT_TOKEN }}
-          commit-message: Update the browser version
+          branch: create-pull-request/browser-update
+          delete-branch: true
           committer: Browser Automation Bot <browser-automation-bot@google.com>
           author: Browser Automation Bot <browser-automation-bot@google.com>
+          commit-message: 'chore: update the pinned browser version'
+          title: 'chore: update the pinned browser version'
+          body: 'Automatically generated by https://github.com/GoogleChromeLabs/chromium-bidi/blob/main/.github/workflows/update-browser-version.yml'
```

Comment:
https://github.com/GoogleChromeLabs/chromium-bidi/pulls?q=is%3Apr+dependabot+is%3Aclosed+label%3Adependencies

Let's also add

```
labels: dependencies
```

to mirror dependabot

Reply:
Added